src/app/transfer-funds/transfer-funds.component.ts

```diff
@@ -340,6 +341,14 @@ export class TransferFundsComponent implements AfterContentInit, OnInit {
     this.identityService.clearJWT();
   }
 
+  /**
+   * Amount in existing committed tips to be fulfilled, in minor units (i.e. pence),
+   * currently just for GBP / UK bank transfers. 0 if donor's not yet loaded.
+   */
+  get pendingTipBalance() {
```

Comment:
I feel like probably we should have explicit type here because otherwise you have to look a few references deep to find the type of 'gbp'. Could be persuade otherwise though.
```suggestion
  get pendingTipBalance(): number {
```